index.html

```diff
@@ -65,20 +65,20 @@
     </linkedin>
     </social_icons>
 
-    <headline>
-        <primary_text>Hey There, I'm Alex.</primary_text>
-        <supporting_text>I'm a Software Developer.</supporting_text>
-        <description>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</description>
+    <headline id="headline">
+        <primary_text id="primary_text">Hey There, I'm Alex.</primary_text>
+        <supporting_text id="supporting_text">I'm a Software Developer.</supporting_text>
+        <description id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</description>
     </headline>
 
-    <illustration_header>
+    <illustration_header id="illustration_header">
         
-        <illustration>
+        <illustration id="illustration">
             <img src="./assets/Illustration-header.svg" alt=""> </img>
         </illustration>
     </illustration_header>
 
-    <base1></base1>
+    <base1 id="base1"></base1>
```

Comment:
- [ ] Kindly I recommend you to use `section` tag for headline section instead of `div` tag because It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

Reply:
headline* tags changed to section.